src/components/OmouComponents/AddSessions.js

```diff
@@ -1,7 +1,7 @@
 import {useDispatch, useSelector} from "react-redux";
 import React, {useMemo, useState} from "react";
 import {bindActionCreators} from "redux";
-import * as registrationActions from "../actions/registrationActions";
+import * as registrationActions from "../../actions/registrationActions";
```

Comment:
this is why we have absolute imports

Reply:
lol my IDE breaks with absolute imports. It can only automatically refactor files if its a relative import, so i had to manually fix all of the absolute imports :(